Makefile

```diff
@@ -225,10 +225,12 @@ QEMU_ACCEL_Y_Linux_amd64=-machine q35,accel=kvm,usb=off,dump-guest-core=off -cpu
 # -machine virt,gic_version=3
 QEMU_ACCEL_Y_Darwin_arm64=-machine $(QEMU_DEFAULT_MACHINE)accel=hvf,usb=off -cpu host
 QEMU_ACCEL_Y_Linux_arm64=-machine virt,accel=kvm,usb=off,dump-guest-core=off -cpu host
+QEMU_ACCEL_Y_$(UNAME_S)_COMMON=
 QEMU_ACCEL__$(UNAME_S)_arm64=-machine virt,virtualization=true -cpu cortex-a57
 QEMU_ACCEL__$(UNAME_S)_amd64=-machine q35 -cpu SandyBridge
 QEMU_ACCEL__$(UNAME_S)_riscv64=-machine virt -cpu rv64
-QEMU_ACCEL:=$(QEMU_ACCEL_$(ACCEL:%=Y)_$(UNAME_S)_$(ZARCH))
+QEMU_ACCEL__$(UNAME_S)_COMMON=-device virtio-gpu-pci
```

Comment:
`virtio-gpu-pci`? I think you meant `virtio-rng-pci`...

Reply:
And that is what you get for letting copilot fill it in. I moved it over, copilot got ahead of me, and my eyes saw the same thing.

Thanks, solid catch. Fixed.